ui/entity/EntityForm.tsx

```diff
@@ -87,15 +104,28 @@ const EntityForm = ({
         onSubmit={ handleSubmit(onFormSubmit) }
         onChange={ handleFormChange }
       >
+
         <Grid
           as="section"
           columnGap="30px"
           rowGap={{ base: 2, lg: 5 }}
           templateColumns={{ base: '1fr', lg: 'minmax(auto, 680px) minmax(0, 340px)' }}
         >
-          <EntityFieldData hint="Choose between uploading a file or entering text data for your entity"/>
 
-          <EntityFieldBtl hint="Block to Live - number of blocks until this entity expires"/>
+          <EntityFieldData hint="Choose between uploading a file or entering text data for your entity"/>
+          <EntityFormRow>
+            <FormFieldText
```

Comment:
When the current date is set and time passes so it’s in the past, it tries to create an entity with negative BTL. Please add a validator that runs on submit to check if the date is not in the past.